server/schedule/operator/kind.go

```diff
@@ -24,22 +24,23 @@ type OpKind uint32
 
 // Flags for operators.
 const (
-	// Include leader transfer.
-	OpLeader OpKind = 1 << iota
-	// Include peer addition or removal. This means that this operator may take a long time.
-	OpRegion
+	// Initiated by merge checker or merge scheduler. Note that it may not include region merge.
+	// the order describe the operator's producer and is very helpful to decouple scheduler or checker limit
+	OpMerge OpKind = 1 << iota
+	// Initiated by range scheduler.
+	OpRange
+	// Initiated by replica checker.
+	OpReplica
 	// Include region split. Initiated by rule checker if `kind & OpAdmin == 0`.
 	OpSplit
-	// Initiated by admin.
-	OpAdmin
 	// Initiated by hot region scheduler.
 	OpHotRegion
-	// Initiated by replica checker.
-	OpReplica
-	// Initiated by merge checker or merge scheduler. Note that it may not include region merge.
-	OpMerge
-	// Initiated by range scheduler.
-	OpRange
+	// Include peer addition or removal. This means that this operator may take a long time.
+	OpRegion
+	// Include leader transfer.
+	OpLeader
+	// Initiated by admin.
+	OpAdmin
```

Comment:
should Admin with higher priority?

Reply:
LGTM, `OpAdmin` should not occupy the limit.

Reply:
It will make `op show admin` cannot be used.

Reply:
> It will make `op show admin` cannot be used.

No, it uses `GetOperatorsOfKind`.

Reply:
Ok